release/tasks/modify-windows-iso-file/modify-windows-iso-file.yaml

```diff
@@ -52,7 +52,9 @@ spec:
         guestfish -a ${TARGET_IMG_FILE_PATH} -m /dev/sda tar-out / - | tar xvf - -C ${ISO_FILES_PATH} -m --no-overwrite-dir --owner=$(id -u) --group=$(id -g) --no-same-permissions
 
         if [ ! -f ${EFI_BOOT}/efisys_noprompt.bin ] || [ ! -f ${EFI_BOOT}/cdboot_noprompt.efi ]; then
-          echo "${EFI_BOOT}/efisys_noprompt.bin or ${EFI_BOOT}/cdboot_noprompt.efi not found in the iso file! Exiting"
+          echo "${EFI_BOOT}/efisys_noprompt.bin or ${EFI_BOOT}/cdboot_noprompt.efi not found in the iso file! Task expects that ISO file contains bootloader without prompt. If the no prompt bootloader is not present, the installation might behave unexpectedly. Exiting"
+          echo "ISO file checksum:" $(sha256sum ${TARGET_IMG_FILE_PATH})
```

Comment:
What does the checksum help with in this case?

Reply:
If anyone reports an issue we can easily check if the iso is from official Microsoft distrubution or if it is some custom iso

Reply:
Is there a public list of checksums mapping to ISOs available?

Reply:
Yes, there are unofficial pages, which contain list of iso checksums and some checksums can be foud in official Microsoft forums

Reply:
Not ideal, but better than nothing.